libs/vkd3d/vkd3d_private.h

```diff
@@ -1335,6 +1348,10 @@ struct d3d12_command_list
     size_t init_transitions_size;
     size_t init_transitions_count;
 
+    struct vkd3d_query_range *query_ranges;
```

Comment:
Are these zero-initialized on creation?

Reply:
Should be since we memset the entire struct at some point.